app/models/article.rb

```diff
@@ -374,6 +374,20 @@ def tag_keywords_for_search
     tags.pluck(:keywords_for_search).join
   end
 
+  def edited?
+    edited_at.present?
+  end
+
+  def readable_edit_date
```

Comment:
I think `readable_edit_date` fits more into decorator than a model. As I see, `article` in the `_dashboard_article` partial should be decorated already.
I would say the same about the `readable_publish_date`, but it needs additional checks on where it's used, and it's probably not the topic of this pr.

Reply:
Since we didn't flag this initially, I didn't move it. Also I didn't want to change the original author's PR. But yeah, we might want to do this in this or another PR :)

Reply:
@lightalloy if you agree we can move both in a decorator once this is merged

Reply:
@rhymes of course, that makes sense.

Reply:
@lightalloy ok cool, I'm deferring the review to you because I've participated in the PR :D